proposals/2025-06-23-tsc-voting-procedures.md

```diff
@@ -0,0 +1,177 @@
+# TSC Voting Procedures Proposal
+
+## Summary
+
+This proposal establishes standardized voting procedures for the Hiero Technical Steering Committee (TSC) to improve decision-making efficiency while maintaining transparency and adherence to charter requirements.
+
+## Motivation
+
+Sometimes the TSC struggles to achieve qourum around key decisions or requires additional time outside of a meeting to deliberate on a decision. Asynchronous voting through Github Pull Requests would enable both flexibility and a solution for voting outside of the short meeting window.  
+
+## Proposal Details
+
+## Voting Methods
+
+### 1. In-Meeting Votes
+
+- **When Used**: For decisions during TSC meetings with quorum present
+- **Threshold**: Simple majority of members present
+- **Recording**: Documented in meeting minutes with vote counts
+
+### 2. Asynchronous Votes via GitHub Pull Request
+
+- **When Used**:
+  - When [quorum](https://github.com/hiero-ledger/hiero/blob/main/technical-charter.md?plain=1#L206) cannot be achieved in meetings
+  - When voting must be completed before next TSC meeting
+  - When a TSC member requests async voting due to inability to attend meeting
+  - For charter-mandated votes requiring 2/3 majority [per Charter section 4. iii](https://github.com/hiero-ledger/hiero/blob/main/technical-charter.md?plain=1#L209)
+- **Location**: Pull requests in the `proposals/` directory
+- **Notification**: All TSC members must be notified via:
+  - GitHub PR mention (@username)
+  - Slack notification in TSC channel
+
+## Voting Process for In-Meeting Votes
+
+### Step 1: Proposal Introduction
+- Present the proposal during TSC meeting
+- Allow for discussion and clarification
+
+### Step 2: Call for Vote
+- TSC Chair or designee calls for vote
+- Confirm [quorum](https://github.com/hiero-ledger/hiero/blob/main/technical-charter.md?plain=1#L206) is present
+
+### Step 3: Voting
+- Members vote verbally or via meeting platform
+- Record each member's vote
+
+### Step 4: Resolution
+- Tally votes and announce result
+- Document in meeting minutes with vote counts
+
+## Voting Process for Asynchronous Votes
+
+### Step 1: Discussion (Optional)
+
+- Create a GitHub Discussion for complex proposals requiring discussion
+- Label with "tsc-discussion"
+- No formal timeline - continue until consensus on proposal language
+
+### Step 2: Proposal Creation
+
+1. Create a new file in `proposals/` directory following the template
+2. File naming convention: `YYYY-MM-DD-brief-description.md`
+3. Open a Pull Request with the finalized proposal
+4. **Proposal is immutable once PR is opened** - no changes allowed
```

Comment:
If we simply clear all approvals if the file is changed, would this also work?

Reply:
> If we simply clear all approvals if the file is changed, would this also work?

I was looking at it from a strictly immutable lens, but of course the Github configuration settings could get in the way here. If discussion is required, it could happen outside of the PR per step 1. If changes are required that were not accurately reflected in the discussion it should require closing the PR and reopening a new one with a clear history / new proposal, similar to how decentralized governance works today commonly. Once you attribute your tokens to a YES or NO answer, you cannot undo it. 

Again, open to amendment / commits here, the initial flow is just a recommendation